test/api/v3/integration/tasks/challenges/GET_tasks_challenge.id.test.js

```diff
@@ -84,4 +84,35 @@ describe('GET /tasks/challenge/:challengeId', () => {
       });
     });
   });
+
+  it('maintains challenge task order', async () => {
```

Comment:
this test should be in POST-tasks_taskId_move_to? same for the group one

Reply:
sorry, I misread the PR description... anyway should we add a test to check that challenge and group tasks can be moved in POST-tasks_taskId_move_to? since right now it only tests users' tasks

Reply:
Sure, that sounds like a good idea.

Reply:
I think this one slipped through :)